### PR TITLE
Attempt to make sense of reasoning for loss

### PR DESCRIPTION
**Pull request by Naman-Bhalla**
I feel the document seems a bit weird. Adding "when" seems to make better sense of it. Open to suggestions on alternatives.
Thanks

### CODE REVIEW

File: tensorflow/docs_src/programmers_guide/custom_estimators.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ model's loss. This is the
 that will be optimized.
 
 We can calculate the loss by calling @{tf.losses.sparse_softmax_cross_entropy}.
-The value returned by this function will be lowest, approximately 0,
+When the value returned by this function will be lowest, approximately 0,
```
Comment (qlzh727) [inline tensorflow/docs_src/programmers_guide/custom_estimators.md]: How about this:

The value returned by this function will be approximately 0 at lowest, when the probability of the correct class (at index `label`') is near 1.0.

Comment (Naman-Bhalla) [reply tensorflow/docs_src/programmers_guide/custom_estimators.md]: That sounds better. Thanks. 
Should I update the PR with it ?

Comment (MarkDaoust) [reply tensorflow/docs_src/programmers_guide/custom_estimators.md]: I just patched it.

**Review by MarkDaoust — APPROVED** [review]
Thanks helping us get this fixed.
